helm/fcp-fd-comms/templates/config-map.yaml

```diff
@@ -1,6 +1,14 @@
 {{- include "ffc-helm-library.container-config-map" (list . "fcp-fd-comms.config-map") -}}
 {{- define "fcp-fd-comms.config-map" -}}
 data:
+  MESSAGE_QUEUE_HOST: {{ .Values.container.messageQueueHost | quote }}
+  {{- if and (.Values.environmentCode) (not .Values.pr) }}
+  MESSAGES_SUBSCRIPTION_ADDRESS: {{ .Values.container.messagesSubscriptionAddress | quote }}-{{ .Values.environmentCode }}
+  MESSAGES_TOPIC_ADDRESS: {{ .Values.container.messagesTopicAddress | quote }}-{{ .Values.environmentCode }}
```

Comment:
When we change environment variables we need to make sure this config map has been updated to reflect the changes.

Reply:
Done.